CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+v1.8.0-RC2/ 2020-03-13
```

Comment:
2020-03-12?

Reply:
Our docker hub reports it was built 4 days ago
https://hub.docker.com/layers/openebs/cstor-istgt/1.8.0-RC2/images/sha256-f7e09b9d698211fdacfd18daee3c5ec0fad08b2cbe92696011a07c029296b61f?context=explore

Reply:
this is about the tag not about the image, So the date is when this tag was created and all the changes done in this tag.